extensions/wikia/AdEngine/js/tracking/adInfoListener.js

```diff
@@ -0,0 +1,167 @@
+/*global define, JSON*/
+define('ext.wikia.adEngine.tracking.adInfoListener',  [
+	'ext.wikia.adEngine.adContext',
+	'ext.wikia.adEngine.lookup.services',
+	'ext.wikia.adEngine.tracking.adInfoTracker',
+	'wikia.log',
+	'wikia.window'
+], function (adContext, lookupServices, tracker, log, win) {
+	'use strict';
+
+	var logGroup = 'ext.wikia.adEngine.tracking.adInfoListener',
+		enabledSlots = {
+			TOP_LEADERBOARD: true,
+			TOP_RIGHT_BOXAD: true,
+			PREFOOTER_LEFT_BOXAD: true,
+			PREFOOTER_MIDDLE_BOXAD: true,
+			PREFOOTER_RIGHT_BOXAD: true,
+			LEFT_SKYSCRAPER_2: true,
+			LEFT_SKYSCRAPER_3: true,
+			INCONTENT_BOXAD_1: true,
+			INCONTENT_PLAYER: true,
+			BOTTOM_LEADERBOARD: true,
+			MOBILE_TOP_LEADERBOARD: true,
+			MOBILE_BOTTOM_LEADERBOARD: true,
+			MOBILE_IN_CONTENT: true,
+			MOBILE_PREFOOTER: true
+		};
+
+	function isEnabled() {
+		return adContext.getContext().opts.enableAdInfoLog;
+	}
+
+	function parseParameters(content) {
```

Comment:
maybe we could use `querystring` module? It's pretty heavily tested by unittests and on production ;)

Reply:
Absolutely! Thanks!